src/docs/08-operation-guide/index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Operation Guide
+permalink: /docs/operation-guide
+---
+
+# Operation Guide
+
+## Overview
```

Comment:
You probably don't need this subtitle here if it's the only section in this page.